content/docs/accessibility.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: accessibility
-title: Accessibility
+title: Accesibilidad
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## ¿Por qué Accesibilidad?
 
-Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.
+La accesibilidad web (también conocida como [** a11y **] (https://en.wiktionary.org/wiki/a11y)) es el diseño y la creación de sitios web que pueden ser utilizados por todos. El soporte de accesibilidad es necesario para permitir que la tecnología de asistencia interprete las páginas web.
```

Comment:
Remove those spaces, all the document is in bold now 😅 
```suggestion
La accesibilidad web (también conocida como [**a11y**] (https://en.wiktionary.org/wiki/a11y)) es el diseño y la creación de sitios web que pueden ser utilizados por todos. El soporte de accesibilidad es necesario para permitir que la tecnología de asistencia interprete las páginas web.
```

Reply:
Done!